services/metadactyl-clj/src/metadactyl/user.clj

```diff
@@ -17,9 +15,9 @@
   (log/debug user-attributes)
   (let [uid (user-attributes :user)]
     (if (empty? uid)
-      (throw+ {:error_code common-errors/ERR_NOT_AUTHORIZED,
-               :user (dissoc user-attributes :password),
-               :message "Invalid user credentials provided."}))
+      (throw+ {:type :clojure-commons.exception/bad-request-field
```

Comment:
Why change the response from `ERR_NOT_AUTHORIZED` to `clojure-commons.exception/bad-request-field`? Isn't this going to change the response status code from a 401 to a 400? I would think a 401 is more appropriate here.